projects/gloo/pkg/utils/validation/proxy_validation.go

```diff
@@ -303,6 +319,41 @@ func GetListenerError(listener *validation.ListenerReport) []error {
 	return errs
 }
 
+func GetListenerWarning(listener *validation.ListenerReport) []string {
+	var warnings []string
+
+	if warning := GetListenerWarn(listener); warning != nil {
```

Comment:
These names confuse me given that this is basically the same name. 
Can we either change its name to be more descriptive or minimally have comments noting why GetListenerWarning is different from getlistenerwarn

Reply:
I'll add comments since I've copied the naming convention already in place and wouldn't wanna change exported methods